pynamodb/connection/base.py

```diff
@@ -1198,11 +1199,39 @@ def query(self,
         """
         operation_kwargs = {TABLE_NAME: table_name}
         name_placeholders = {}
+        expression_attribute_values = {}
+
+        tbl = self.get_meta_table(table_name)
+        if tbl is None:
+            raise TableError("No such table: {0}".format(table_name))
+        if index_name:
+            hash_keyname = tbl.get_index_hash_keyname(index_name)
+            if not hash_keyname:
+                raise ValueError("No hash key attribute for index: {0}".format(index_name))
+        else:
+            hash_keyname = tbl.hash_keyname
+
+        key_condition_expression = self._get_condition_expression(table_name, hash_keyname, '__eq__', hash_key)
+        if key_conditions is None or len(key_conditions) == 0:
+            pass  # No comparisons on sort key
+        elif len(key_conditions) > 1:
+            raise ValueError("Multiple attributes are not supported in key_conditions: {0}".format(key_conditions))
+        else:
+            (key, condition), = key_conditions.items()
```

Comment:
```python
(key, condition), _ = key_conditions.items()  # this explicitly documents that you are aware of the second part of the tuple and in fact do not want it. otherwise people may think it's a stray comma
```